managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+    meltano add loader target-postgres
+    ```
+
+#### Configuring the extractor to connect to your existing database
+
+1. Run the command below to configure the connection to your existing database:
+
+    ```sh
+    meltano config tap-postgres set --interactive
+    ```
+
+    A list of settings displays, with a number associated with each parameter.
+
+2. Fill in the required information below by entering the number associated with each parameter:
+
+    - **Database Name** (5)
+    - **Host** (13)
+    - **Password** (14)
+    - **Port** (15)
+    - **Username** (32)
+
+3. Select the tables you want to load:
+
+    ```
+    meltano select tap-postgres {schema-name}-{table1name}
+    meltano select tap-postgres {schema-name}-{table2name}
+    ...
+    ```
+
+    <Message type="tip">
+    For instance `meltano select tap-postgres public-cities` if you are importing the table `cities` from the schema `public`.
+    You can check the selected tables and columns with `meltano select tap-postgres --list --all`.
+    </Message>
+
+#### Configuring the loader to connect to your new Managed Database for PostgreSQL
+
+1. Run the command below to configure the connection to your existing database:
+
+    ```sh
+    meltano config tap-postgres set --interactive
+    ```
+
+    A list of settings displays, with a number associated with each parameter.
+
+2. Fill in the required information below by entering the number associated with each parameter:
+
+    - **Database Name** (5)
+    - **Host** (13)
+    - **Password** (14)
+    - **Port** (15)
+    - **Username** (32)
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+#### Executing the data import and loading
+
+1. Run the following command to execute the data import and loading:
+
+    ```sh
+    meltano run tap-postgres target-postgres
+    ```
+
+2. When finished, make sure your data is stored in your new database by connecting to it, and performing a query.
+
+    <Message type="note">
+    By default, the `tap-postgres` extractor loads data in the `tap_postgres` schema, rather than the `public` schema. You can find your loaded tables with the following command:
+
+    ```
+    SELECT * from tap-postgres.{table1name} LIMIT 10;
+    ```
+    </Message>
+
+## Importing data from an existing PostgreSQL backup
+
+### Using pg_restore
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

Comment:
We have to adapt a little bit to say that the user must have installed PostgreSQL with pg_restore in the same version as the one of their Managed PostgreSQL